plugins/pulp_docker/plugins/distributors/distributor_export.py

```diff
@@ -92,9 +92,12 @@ def validate_config(self, repo, config, config_conduit):
         :return: tuple of (bool, str) to describe the result
         :rtype:  tuple
         """
+        repo = model.Repository.objects.get(repo_id=repo_transfer.id)
+        repo.repo_id = repo_transfer.id
```

Comment:
Is this necessary? Shouldn't that field be populated already? I see the same pattern in other parts of this PR.